pyanaconda/core/configuration/payload.py

```diff
@@ -61,6 +61,19 @@ def enable_closest_mirror(self):
         """
         return self._get_option("enable_closest_mirror", bool)
 
+    @property
+    def enable_repositories_from_treeinfo(self):
+        """Enable installation from .treeinfo extra repos.
+
+        This flag controls whether or not Anaconda will automatically
+        enable any repos defined in .treeinfo for your product.
+
+        The use of additional repos from .treeinfo is disabled by default.
+
+        The BaseOS repo is always enabled.
```

Comment:
I would change it to `The base repo is ...`. I think that Fedora's base repo has a different name.

Reply:
I'll make this change.